src/Payum/Core/Tests/Bridge/Propel/Storage/Propel1StorageTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Payum\Core\Tests\Bridge\Propel\Storage;
+
+use Payum\Core\Bridge\Propel\Storage\Propel1Storage as PropelStorage;
+use Payum\Core\Tests\Mocks\Model\PropelModel;
+
+class Propel1StorageTest extends \PHPUnit_Framework_TestCase
+{
+    public static function setUpBeforeClass()
+    {
+        if (false == class_exists('Propel', $autoload = true)) {
```

Comment:
this is not really needed since you added propel as dependency to `require-dev`. That's enough.